ext/DynaLoader/DynaLoader_pm.PL

```diff
@@ -292,12 +298,14 @@ sub croak   { require Carp; Carp::croak(@_)   }
 
 sub bootstrap_inherit {
     my $module = $_[0];
-    local *isa = *{"$module\::ISA"};
-    local @isa = (@isa, 'DynaLoader');
+
+    no strict 'refs';
+    push @{"$module\::ISA"}, 'DynaLoader';
```

Comment:
I consider this push to be the riskiest part of this change. I'd love someone to confirm the change is ok.

Reply:
This changes that module's `@ISA` permanently, rather than only while running `bootstrap`. There's also the chance that something is relying on `@Dynaloader::isa` being set while running `bootstrap`, this seems unlikely but I'm not that familiar with how this module works. The original code should be fine with `no strict 'refs'` and declaring `our @isa`.

Reply:
Agree on all counts with @Grinnz, he's too quick.

Reply:
Yeah, the `local` here is absolutely essential. We only want the modules to inherit from `DynaLoader` during bootstrap, so that it has a default `dl_load_flags` implementation during that bootstrap. This design is unfortunate, but no one really knew how to OO back in 1994.

Reply:
ok. I put it back.